email-filters.html

```diff
@@ -0,0 +1,170 @@
+            </a>
+            <button class="navbar-toggler toggler-home" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse " id="navbarSupportedContent">
+                <ul class="navbar-nav ml-lg-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="index.html">Home <span class="sr-only">(current)</span>
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="features.html">Features</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="security.html">Security</a>
+                    </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="modules.html">Mods</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="license.html">License</a>
+                    </li>
+                    <li class="nav-item">
+                        <!--<a class="nav-link" href="tests.html">Test</a>-->
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="documentation.html">Documentation</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://unencumberedbyfacts.com" target="_blank">Blog</a>
+                    </li>
+                    <li class="nav-item">
+                        <a onclick="document.getElementsByClassName('navbar-collapse')[0].style.display='none';"
+                           class="nav-link" href="contribute.html">Contribute</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+
+<section class="content-section container">
+    <h2>Email filters</h2>
+    <p>Cypht now supports Sieve, a powerful email filtering language that gives you complete control over your inbox. With Sieve, you can create complex filters to sort, move, delete, and flag your emails automatically. You can also use Sieve to create custom notifications and alerts.</p>
+    <p>Here are just a few of the things you can do with Sieve:</p>
+    <hr>
+    <ul>
+        <li>
+            Filter out spam and unwanted emails. Sieve filters can be used to filter out spam and unwanted emails based on their sender, subject, or content.
+        </li>
+        <hr>
+        <li>
+            Organize your inbox automatically. Sieve filters can be used to automatically sort your emails into different folders or labels, based on their sender, subject, or content.
```

Comment:
"sort" is ambiguous. It could make people think they can change the sort order.

Also, labels are still on todo: https://github.com/cypht-org/cypht/issues/13

I suggest to change from:
"Sieve filters can be used to automatically sort your emails into different folders or labels, based on their sender, subject, or content."

to:
"Sieve filters can be used to automatically move or copy your emails into different folders based on their sender, subject, or content."